coordinator/src/position/models.rs

```diff
@@ -197,6 +202,84 @@ impl Position {
             trade_params.average_execution_price(),
         )
     }
+
+    #[must_use]
```

Comment:
It just means that the compiler will complain if we do not use the return value of this function. Since this is a pure function that returns `Self`, it doesn't do anything if you don't use the return value.

In an earlier version, I ran into a bug precisely because I forgot to use the returned `Position`.